test/bluesky.auth.flow.test.ts

```
@@ -0,0 +1,131 @@
+      if (url.startsWith('https://as.example/par')) {
+        parCalls += 1;
+        if (parCalls === 1) {
+          return new Response('{}', {
+            status: 400,
+            headers: {
+              'WWW-Authenticate': 'Bearer error="use_dpop_nonce"',
+              'DPoP-Nonce': 'server-par-nonce'
+            }
+          });
+        }
+        return Response.json({ request_uri: 'urn:ietf:params:oauth:request_uri:test', expires_in: 90 });
```

ESLint

[error] Replace `·request_uri:·'urn:ietf:params:oauth:request_uri:test',·expires_in:·90` with `⏎··········request_uri:·'urn:ietf:params:oauth:request_uri:test',⏎··········expires_in:·90⏎·······`

(prettier/prettier)